include/seqan3/utility/views/chunk.hpp

```diff
@@ -689,12 +692,20 @@ class chunk_view<urng_t>::basic_iterator : public maybe_iterator_category<maybe_
     //!\brief Move to the end of the next chunk.
     constexpr it_t get_next_end_of_chunk(it_t start_of_chunk) const
     {
+        if constexpr (std::sized_sentinel_for<sentinel_t, it_t>)
+            if (urng_end - start_of_chunk < 0)
+                return current_chunk.end();
+
```

Comment:
hm I don't think I like this solution more. Now we kind of state that we expect that `start_of_chunk` will exceed `urng_end`. Although this is clearly not intended behavior? I think making sure that `start_of_chunk` never exceeds `urange_end` in the first place is a better idea no?